examples/pytorch/vision/Face_Detection/README_M4.md

```diff
@@ -0,0 +1,130 @@
+# Code for Face Detection Experiments with RNNPool
+## Requirements
+1. Follow instructions to install EdgeML operators and their pre-requisites [here](https://github.com/microsoft/EdgeML/blob/master/pytorch/README.md).
+2. Install requirements for face detection model using
+``` pip install -r requirements.txt ``` 
+We have tested the installation and the code on Ubuntu 18.04 with Python 3.6, Cuda 10.2 and CuDNN 7.6
+
+## Dataset - WIDER Face
+1. Download WIDER face dataset images and annotations from http://shuoyang1213.me/WIDERFACE/ and place them all in a folder with name 'WIDER_FACE'. That is, download WIDER_train.zip, WIDER_test.zip, WIDER_val.zip, wider_face_split.zip and place it in WIDER_FACE folder, and unzip files using: 
+
+```shell
+cd WIDER_FACE
+unzip WIDER_train.zip
+unzip WIDER_test.zip
+unzip WIDER_val.zip
+unzip wider_face_split.zip
+cd ..
+
+```
+
+2. Set environment variable DATA_HOME to the parent directory of the above folder
+That is, if the WIDER_FACE folder is created in /mnt folder
+
+``` export DATA_HOME='/mnt' ```
+
+Note that for Windows '/' should be replaced by '\'.
+
+
+3. Run
+``` IS_QVGA_MONO=1 python prepare_wider_data.py ```
+
+## Dataset - SCUT Head B
+Download SCUT Head Part B dataset images and annotations from https://github.com/HCIILAB/SCUT-HEAD-Dataset-Release. Unzipping will create a folder by the name 'SCUT_HEAD_Part_B'. Place this folder in the same parent directory as the WIDER_FACE folder.
```

Comment:
WIDER FACE is a single dataset where the type of images in train test and val are similar. SCUT Head Part A and B are different datasets with different distributions. We choose Part B since average number of people in the images is much lesser than in Part A. So I think we can make separate folders in the data directory for both.